examples/portfolio/src/pages/404.astro

```diff
@@ -4,16 +4,16 @@ import Footer from '../components/Footer/index.jsx';
 import Nav from '../components/Nav/index.jsx';
 ---
 
-<html lang="en">
+<html lang='en'>
 	<head>
-		<MainHead title="Not Found" />
+		<MainHead title='Not Found' />
 	</head>
 	<body>
-		<nav />
-		<div class="wrapper mt4 mb4">
+		<nav></nav>
+		<div class='wrapper mt4 mb4'>
 			<h1>Page Not Found</h1>
 			<p>Not found</p>
 		</div>
-		<footer />
+		<footer></footer>
 	</body>
```

Comment:
This is unrelated to the formatting changes, but isn't this file supposed to use the imported components? 😅